apps/explorer/components/Contract/index.tsx

```diff
@@ -274,39 +289,40 @@ export function Contract({ contract, rates, renewedFrom, renewedTo }: Props) {
   )
 }
 
-function isProperlyFormedNewContract(contract: SiaCentralContract) {
+function isProperlyFormedNewContract(contract: ExplorerFileContract) {
   // renter payout, host payout
-  if (contract.valid_proof_outputs?.length !== 2) {
+  if (contract.fileContract.validProofOutputs?.length !== 2) {
     return false
   }
   // renter payout, host payout, and host burned
-  if (contract.missed_proof_outputs?.length !== 3) {
+  if (contract.fileContract.missedProofOutputs?.length !== 3) {
     return false
   }
 
   const { renterPayoutValid, renterPayoutMissed } =
     getNewContractFormattedOutputs(contract)
 
   // renter payout valid and missed should be the same
-  if (renterPayoutValid.value !== renterPayoutMissed.value) {
+  if (renterPayoutValid?.value !== renterPayoutMissed?.value) {
+    // Does we need to catch undefined cases now?
```

Comment:
I believe the earlier check does ensure these will be defined, although TypeScript is not smart enough to provide that here.